src/Player.js

```diff
@@ -1,5 +1,15 @@
 /* Represents a player in the game; either computer-controlled or human-controlled */
 class Player 
 {
+	constructor(ownedPieces, capturedPieces)
```

Comment:
For output's sake would we want to include a name or a tag as well? Is this also where we would pass in a direction?

Reply:
Direction, yes. Name, no, because everything actually needs names and I added that as part of the Parser task (since that's the first time we'll need to identify things).